src/test/java/RecipeTest.java

```diff
@@ -58,9 +62,8 @@ public void addBeginningStep() {
         new Step("Put the cheese in the bread")
```

Comment:
I know you had this from before, but you could consider initializing the expected with "Index 0!" and then use List.addAll() : https://docs.oracle.com/javase/8/docs/api/java/util/List.html#addAll-java.util.Collection-